### ISSUE TITLE
image2petscii tries to load a nonexisting config

### ISSUE THREAD

**Issue by schorsch3000** [micheldebree/c64-tools#9]
I've installed image2petscii via `npm i -g image2petscii`

running it results in the following error:

```
$ img2petscii -b optimal out/000000001.png 
node:internal/errors:490
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/local/lib/node_modules/img2petscii/built/config.js' imported from /usr/local/lib/node_modules/img2petscii/built/cli.js
    at new NodeError (node:internal/errors:399:5)
    at finalizeResolution (node:internal/modules/esm/resolve:326:11)
    at moduleResolve (node:internal/modules/esm/resolve:945:10)
    at defaultResolve (node:internal/modules/esm/resolve:1153:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:838:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

am i doing something wrong?

**Comment by micheldebree** [comment]
Sorry about that I must have broken something. I am currently on holiday and unable to look into it. Maybe installing an older version will help. My internet access is limited so I cannot be or much help right now, I will look into it when I get back.

**Comment by schorsch3000** [comment]
that project is on hold anyway for a few weeks, take your time and have relaxing holidays :-)

**Comment by micheldebree** [comment]
I cannot reproduce on my system, but can you check this for me:

- look in /usr/local/lib/node_modules/img2petscii/built
- Is there a file Config.js? Does it start with a capital C?
- If so, rename to config.js, all lowercase
- Try if this helps

I noticed the npm package has a Config.js, while the source has config.js. If this is the problem I will try to fix ASAP

**Comment by schorsch3000** [comment]
jep, that's it :)

**Comment by micheldebree** [comment]
Seems to be fixed in 0.0.5, although I am not sure what went wrong and what fixed it...
